### ISSUE TITLE
Smazani inzerátu

### ISSUE THREAD

**Issue by DenesovaR** [obchodniuspech/uajobs.cz#2]
Dobrý den, chtěla jsem skrýt inzerát, na který mi během 24 hodin přišlo přes 15 odpovědí. Šla jsem tedy do administrace inzerátu a vybrala ve scroll down volbě "smazat" a potvrdila jsem. Bohužel jsem si neuvědomila, že se mi zároveň se smazáním inzerátu smažou i veškeré odpovědi na něj. Jde o : https://uajobs.cz/contact/315/
https://uajobs.cz/contact/315/f9e260a480c6b5c303ee969b16e64f73

Lze prosím toto nějak vrátit zpět ? Případně mi pouze zobrazit odpovědi ale již nezobrazoval inzerát v nabídce jako aktivní ? Mrzelo by mě, kdybych přišla o všechny kontakty na zájemce o práci. Děkuji předem za odpověď. Denešová

**Comment by DenesovaR** [comment]
Inzerát jsem znovu odeslala ke schválení. Doufám, že link na odpovědi zůstane stejný a zobrazí se mi i předchozí odpovědi s kontakty.

**Comment by obchodniuspech** [comment]
Dobrý den,  inzerát se maže jako celek. Ale data archivujeme. Zřejmě jsem s Vámi komunikoval již skrze email, prosím nesdílejte zde konkrétní odkazy, vzhledem k tomu, že jsou v podstatě heslem k osobním údajům zájemců. Odkaz jsem Vám tedy změnil a zaslal nový.

**Comment by DenesovaR** [comment]
Ano, už vyřešeno přes email. Děkuji. Odkazy jsem smazala z komentáře výše. Děkuji za upozornění.
